### ISSUE TITLE
Show something on unconnected ports

### ISSUE THREAD

**Issue by nomeata** [nomeata/incredible#24]
I.e. the proposition that would be there if one had a connection with nothing else there.

Could use the same display style as for the prototypes.

**Comment by nomeata** [comment]
Marked as a UI bug, but this will require some adjustments in the logic, i.e. return a map from port description to proposition. When this is done, it might make sense to return only the status of each connection, as the label for the connection can be fetched from map just mentioned.

**Comment by gattschardo** [comment]
The port to prop map is something that is also needed for the rule export, see 315f8f80394c37ed50bb610fb93fcb4cb58b5c46, `Analysis.prepare` returns the `BlockProps` map which contains the relevant info.

**Comment by nomeata** [comment]
Nice. Although I would expect type `Map PortSpec Proposition` there, as assumption and conclusions also need a proposition at their port. OTOH, I consider adding them as blocks as well, in which case `PortSpec` becomes isomorphic to `(Key Block, Key Port)`, in which case we are almost back at what we have right now. All in all: Either works :-)

**Comment by gattschardo** [comment]
> assumption and conclusions also need a proposition at their port.

Do they? They have their proposition written on them anyways, so if nothing is connected, there is no extra information to display on their ports, right?

**Comment by nomeata** [comment]
Yes, but the fact that the proposition written on them is, in a way, a deviation from the usual. If I knew what else I could put on there, I’d rather do that :-)

**Comment by NicoleRauch** [comment]
Maybe "START" and "GOAL"? But I'd also prefer to directly see the propositions written on them. It's like these blocks are good for providing these propositions, and the other blocks are good for providing some "adaptors" or something. So I don't see a real deviation there.

**Comment by nomeata** [comment]
Also wanted by one of the ITP reviewers.
